mysql-test/main/create_trigger_on_schedule.result

```diff
@@ -0,0 +1,59 @@
+SET timestamp=UNIX_TIMESTAMP('2014-11-01 10:20:30');
+SET GLOBAL event_scheduler=off;
+CREATE TABLE t1 (a INT);
+CREATE OR REPLACE TRIGGER IF NOT EXISTS ev1 ON SCHEDULE EVERY 1 SECOND DO DROP DATABASE db1;
+ERROR HY000: Incorrect usage of OR REPLACE and IF NOT EXISTS
+CREATE OR REPLACE TRIGGER IF NOT EXISTS ev1 IN SCHEDULE EVERY 1 SECOND DO DROP DATABASE db1;
```

Comment:
That's not very useful, is it? The parser aborts at `IF NOT EXISTS` it doesn't even read your `IN`.

Reply:
Hi, sorry about that. Actually, I noticed the error was a odd, but I thought maybe it was intended. I updated the command, now showing a parser error. Does it look good now?

Reply:
Not really. Now you have
```sql
CREATE OR REPLACE EVENT ev1 IN SCHEDULE EVERY 1 SECOND DO DROP DATABASE db1;
```
which correctly fails with a syntax error, as it always did. You implemented `CREATE TRIGGER` but you test here `CREATE EVENT`, old syntax.

Reply:
Ahh, true. I don't know how didn't saw that or even how `EVENT` appeared there in the first place, I'm sure I was using trigger. :laughing: